internal/gps/result.go

```diff
@@ -79,7 +79,10 @@ func WriteDepTree(basedir string, l Lock, sm SourceManager, sv bool, logger *log
 			}
 
 			if sv {
-				filepath.Walk(to, stripVendor)
+				err := filepath.Walk(to, stripVendor)
+				if err != nil {
+					errCh <- errors.Wrapf(err, "failed to strip vendor from %s", p.Ident().ProjectRoot)
+				}
```

Comment:
Now the `errCh` buffer is too small since each routine could send 2 errors.  Does that even make sense? Or should the first error terminate the goroutine? (maybe defer the `wg.Done()` so we can return from there)